src/app/shared/components/playlist-viewer/playlist-viewer.component.ts

```diff
@@ -30,9 +30,9 @@ export class PlaylistViewerComponent implements OnInit {
   @Output() queuePlaylist = new EventEmitter<GoogleApiYouTubePlaylistResource>();
   @Output() playPlaylist = new EventEmitter<GoogleApiYouTubePlaylistResource>();
   @Output() queueVideo = new EventEmitter<GoogleApiYouTubeVideoResource>();
-  @Output() playVideo = new EventEmitter<GoogleApiYouTubeVideoResource>();
+  @Output() playVedio = new EventEmitter<GoogleApiYouTubeVideoResource>();
```

Comment:
There seems to be a typo in the variable name `playVedio`. It should be `playVideo`.